views/js/qtiCreator/widgets/static/img/Widget.js

```diff
@@ -25,7 +23,7 @@ define([
         //check file exists:
         inlineHelper.checkFileExists(this, 'src', options.baseUrl);
         $('#item-editor-scope').on('filedelete.resourcemgr.' + this.element.serial, function(e, src){
-            if(itemUtil.fullpath(img.attr('src'), baseUrl) === itemUtil.fullpath(src, baseUrl)){
+            if (this.getAssetManager().resolve(img.attr('src')) === this.getAssetManager().resolve(src)) {
```

Comment:
I think this was one of the last remaining usage of the `itemUtil.fullpath` function which is deprecated. Please verify if it is, and then remove it!